app/components/task-card-assignee-list.js

```diff
@@ -0,0 +1,178 @@
+import Ember from 'ember';
+import { getCode } from 'ember-keyboard';
+
+const {
+  Component,
+  computed,
+  computed: { alias, not },
+  get,
+  inject: { service },
+  isEmpty,
+  observer,
+  on,
+  run: { bind, once },
+  set
+} = Ember;
+
+export default Component.extend({
+  classNames: ['select-dropdown', 'task-assignee__users-list'],
+  tagName: 'div',
+  cursorAt: 0,
+  cursorWas: 0,
+  // hidden: false,
+  lastQuery: null,
+  limit: 5,
+  results: [],
+
+  store: service(),
+
+  numberOfResults: alias('results.length'),
+  queryChanged: observer('query', function() {
+    once(this, '_search');
+  }),
+
+  _isNewQuery: not('_sameQuery'),
+  _sameQuery: computed('query', 'lastQuery', function() {
+    return get(this, 'query') === get(this, 'lastQuery');
+  }),
+
+  init() {
+    this._super(...arguments);
+    set(this, 'keyboardActivated', true);
+    set(this, 'boundKeyHandler', bind(this, this.keyHandler));
+  },
+
+  didInsertElement() {
+    this.$(window).on('keydown', this.boundKeyHandler);
+  },
+
+  willDestroyElement() {
+    this.$(window).off('keydown', this.boundKeyHandler);
+  },
+
+  keyHandler(e) {
+    let key = getCode(e);
+    this.send('getKeyDown', key);
+  },
+
+  actions: {
+    getKeyDown(key) {
+      let cursorAt;
+      switch (key) {
+        case 'ArrowDown':
+          cursorAt = get(this, 'cursorAt');
+          this._setPosition(++cursorAt);
+          set(this, 'hidden', false);
+          break;
+        case 'ArrowUp':
+          cursorAt = get(this, 'cursorAt');
+          this._setPosition(--cursorAt);
+          set(this, 'hidden', false);
+          break;
+        case 'Comma':
+        case 'Enter':
+          this._selectAssigneeSkill();
```

Comment:
copy-paste remnant from skill list component, I guess. Can probably remove the section.